src/test/java/com/github/tomakehurst/wiremock/common/DateTimeOffsetTest.java

```diff
@@ -113,10 +113,10 @@ public void shiftsZonedDateTimes() {
   @Test
   public void offsetsProvidedDateByConfiguredAmount() throws Exception {
     DateTimeOffset offset = DateTimeOffset.fromString("3 days");
-    Date startingDate = ISO8601.parse("2018-04-16T12:01:01Z");
+    Date startingDate = STD_DATE_FORMAT.parse("2018-04-16T12:01:01Z");
     Date finalDate = offset.shift(startingDate);
 
-    assertThat(ISO8601.format(finalDate), is("2018-04-19T12:01:01Z"));
+    assertThat(STD_DATE_FORMAT.format(finalDate), is("2018-04-19T12:01:01.000+00:00"));
```

Comment:
This was the only change that had me slightly puzzled, but since it's only in tests I don't think it matters much.

Reply:
This looks like a breaking change to me